govc/events/command.go

```diff
@@ -45,6 +48,8 @@ func (cmd *events) Register(ctx context.Context, f *flag.FlagSet) {
 	cmd.DatacenterFlag.Register(ctx, f)
 
 	f.IntVar(&cmd.Max, "n", 25, "Output the last N events")
+	f.BoolVar(&cmd.Tail, "t", false, "Tail events")
```

Comment:
I'm fine either way -- the reason I went with -t is because -f IMO is more closely associated with file -- so if we wanted push events to a file -- yeah we could pipe that, but that's where my mind was...your call tough guy